forms-web-app/src/controllers/your-details.js

```diff
@@ -0,0 +1,31 @@
+const { appealsApi } = require('../lib/appeals-api-wrapper');
+
+const VIEW = {
+  YOUR_DETAILS: 'your-details/index',
+};
+
+exports.getYourDetails = (req, res) => {
+  res.render(VIEW.YOUR_DETAILS, {
+    appeal: req.session.appeal,
+  });
+};
+
+exports.postYourDetails = async (req, res) => {
+  const body = {
+    appellantName: req.body['appellant-name'],
+    appellantEmail: req.body['appellant-email'],
+  };
+
+  try {
+    const uuid = (req.session && req.session.uuid) || '';
+    const appealJson = await appealsApi(body, uuid);
+
+    req.session.uuid = uuid;
+    req.session.appeal = appealJson;
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.error(e);
```

Comment:
do we have a more preferable way of logging / tracking errors ?

Reply:
I would prefer to use [pino](https://getpino.io) as that gives a nice JSON object and can use [pino-pretty](https://github.com/pinojs/pino-pretty). See [appeals-service-api](https://github.com/foundry4/appeal-planning-decision/blob/master/appeals-service-api/src/lib/logger.js)

Reply:
I believe I have done this in line with the appeals service api implementation, cheers :+1: